data/brands/amenity/fast_food.json

```diff
@@ -816,6 +816,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Butchies",
+      "locationSet": {
+        "include": ["gb-lon.geojson"]
+      },
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Butchies",
+        "brand:wikidata": "Q111015134",
+        "cuisine": "chicken",
+        "diet:vegan": "yes",
```

Comment:
```suggestion
```
Sorry, I don't think this time like  #5801 (doughnuts said that vegan is a key point, but this time it is a **chicken** fast food)

So I don't think this tag is needed. Maybe the case is like #5727

Reply:
Fair play, I will remove the tag in this case.